src/common/blockchain.sh

```diff
@@ -76,15 +76,25 @@ function authenticate_org {
 #   None
 #######################################
 function provision_blockchain {
-    if ! cf service ${BLOCKCHAIN_SERVICE_INSTANCE} > /dev/null 2>&1
+    SERVICE_OUTPUT=$(cf service ${BLOCKCHAIN_SERVICE_INSTANCE})
+    RETVAL=$?
+
+    if [ $RETVAL -eq 0 ] && [[ ${SERVICE_OUTPUT} != *"service: ${BLOCKCHAIN_SERVICE_PLAN}"* ]]
+    then
+        echo "Existing service is not the correct type."
```

Comment:
we can make this message a bit more explicit?  Can we say that name provided for blockchain service exist but it is not an actual blockchain service?  Something in those lines.